### PR TITLE
scripts/install-build-deps: skip Powertools repo installation if exists

### PR DESCRIPTION
**Pull request by madhavemuri**
For Rocky Linux OS "install-build-deps --without-ansible" is exiting
if Powertools repo is not available.
If internal Powertools repo is available installation of packages will be
continued.

Also installs Epel-release repo if accessible.

Signed-off-by: Madhavrao Vemuri <madhav.vemuri@seagate.com>

### CODE REVIEW

File: scripts/install-build-deps [hunk 1]
```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
 if ! $use_ansible ; then
   case $(distro_type) in
       redhat)
-        yum-config-manager --enable powertools
+        source /etc/os-release
+        if [[ $NAME == "Rocky Linux" ]];
+        then
+	    if [[ ! `yum repolist | grep Power` ]]
+            then
```
Comment (andriytk) [inline scripts/install-build-deps:152]: Style: use `if ...; then` on the same line.

File: scripts/install-build-deps [hunk 2]
```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
...
+                yum-config-manager --enable powertools
+                yum -y install epel-release
+            fi
+	elif [[ $NAME == "Oracle Linux Server" ]]
```
Comment (andriytk) [inline scripts/install-build-deps:156]: That's cool, did you check it? Where?

File: scripts/install-build-deps [hunk 3]
```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
...
+	then
+                yum-config-manager --enable ol8_codeready_builder
+        else
+		yum -y install epel-release
+	fi
+	echo "Make sure that epel-release repo is present.."
```
Comment (andriytk) [inline scripts/install-build-deps:162]: Shouldn't this go before 1st if?

Comment (andriytk) [inline scripts/install-build-deps:162]: Shouldn't this go before `source /etc/os-release` line?

**Comment by andriytk** [comment]
@madhavemuri, if team leads don't use that template in PR - why ordinary devs would use it then? And if so - why do we have it at all?

File: scripts/install-build-deps (hunk 2 above)
Comment (madhavemuri) [reply scripts/install-build-deps:156]: yes, epel have to installed separately.

File: scripts/install-build-deps (hunk 3 above)
Comment (madhavemuri) [reply scripts/install-build-deps:162]: This comment is added for Oracle Linux and other OSes.

File: scripts/install-build-deps (hunk 2 above)
Comment (madhavemuri) [reply scripts/install-build-deps:156]: config.vm.box = "generic/oracle8"
  Oracle linux vagrant box is available for public use.

File: scripts/install-build-deps (hunk 3 above)
Comment (madhavemuri) [reply scripts/install-build-deps:162]: > @madhavemuri, if team leads don't use that template in PR - why ordinary devs would use it then? And if so - why do we have it at all?
@andriytk : In this PR i think main description have captured enough details, adding in template form will add overhead in understanding. I do follow the template for most of the PR's.

**Comment by madhavemuri** [comment]
> @madhavemuri, if team leads don't use that template in PR - why ordinary devs would use it then? And if so - why do we have it at all?

@andriytk : In this PR i think main description have captured enough details, adding in template form will add overhead and many not applicable cases are there. I do follow the template for most of the PR's.

File: scripts/install-build-deps (hunk 3 above)
Comment (madhavemuri) [reply scripts/install-build-deps]: Moved it to the required place.

File: scripts/install-build-deps (hunk 1 above)
Comment (madhavemuri) [reply scripts/install-build-deps]: Done.

**Review by mehjoshi — APPROVED** [review]

**Pull request merged**